package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@google/gapic-generator-typescript",
+  "version": "0.0.1",
+  "description": "**This is not an official Google product.**",
```

Comment:
Maybe a better description :)   I would argue this _is_ an official google product.

Reply:
Fixed!